management/connection.go

```diff
@@ -693,7 +693,7 @@ type ConnectionOptionsSMS struct {
 	Provider              *string                          `json:"provider,omitempty"`
 	GatewayURL            *string                          `json:"gateway_url,omitempty"`
 	GatewayAuthentication *ConnectionGatewayAuthentication `json:"gateway_authentication,omitempty"`
-	ForwardRequestInfo    *bool                            `json:"forward_request_info,omitempty"`
+	ForwardReqInfo        *bool                            `json:"forward_req_info,omitempty"`
```

Comment:
We don't need to change the name of the attribute, as that would be a breaking change. We just need to update the json tag.

Reply:
Given that the property wasn't functional, I figured it was worth the breaking change in order to ensure the strict property and API property were aligned

Reply:
The weird thing is that the API actually persists that value and returns it back after in the get as forward_request_info.

Reply:
Yeah, I think it just persists any value. I tested sending a property `foo` and the API accepted and persisted that value